components/logins/android/build.gradle

```diff
@@ -92,5 +98,6 @@ afterEvaluate {
 }
 
 apply from: "$rootDir/publish.gradle"
+apply from: "https://github.com/mozilla-mobile/android-components/raw/v$android_components_version/components/service/glean/scripts/sdk_generator.gradle"
```

Comment:
Not too excited about the circular dependency on a-c here, but looks like we already had it for other reasons.  If we want to avoid this, we could move all the metrics logic into the logins component wrapper in a-c, but ugh...

Reply:
Hey @rfk , have you considered the above comment?

Reply:
I'm not excited about adding these to `support-sync-telemetry` or a-c, either. We moved Sync telemetry collection logic into a-c mostly for expediency, and because we didn't have much experience at the time to know what a better approach would be. And, now that we're also collecting storage telemetry, I'm worried that bolting more onto `s-s-t` will saddle us with additional tech debt, while we're still figuring out the shape of the Rust APIs.

For performance metrics, we'd also need to either expose more implementation details to a-c, or keep them coarse. It's another version of the Sync timing problem that we saw—a-c doesn't have visibility into the queries we run, just like it doesn't have visibility into the sync lifecycle, or the FFI—so it can't collect finer-grained stats beyond "this operation started" and "this operation finished".